### ISSUE TITLE
Inaccuracy In Air Force One

### ISSUE THREAD

**Issue by PeeMoldy** [fgsfdsfgs/perfect_dark#331]
So the door on the floor at the back of the plane is blown off at the start of the level, even before the Skedar Ship attaching to Air Force One cutscene. This is inaccurate, as the door is supposed to be on and openable until that scene happens, then it's blown off. Also the textures of the blown off door look messed up. Can this be fixed in the next build?

**Comment by fgsfdsfgs** [comment]
Can you or somebody else provide screenshots of how it looks in the port and how it's supposed to look?

**Comment by PeeMoldy** [comment]
> Can you or somebody else provide screenshots of how it looks in the port and how it's supposed to look?

I'll try to get screenshot later. It has the red grated trapdoor like Chicago where you reprogram the taxi, or dD Investigation where you take the shortcut to the scientist and K7 Avenger guard.

**Comment by Flax4** [comment]
I quickly ran through the map on "agent" difficulty, I couldn't find any missing door or weird textures.

**Comment by PeeMoldy** [comment]
> I quickly ran through the map on "agent" difficulty, I couldn't find any missing door or weird textures.
> 
> Edit: I also quickly ran through it on a N64 emulator to compare (RMG, using parallel plugin) and it looks the same to me. I could just be blind though since I'm not entirely sure where to look.

Go to the room that Trent Easton runs into after he does his evil laugh, go forward and you'll see a trap door to your right. Start in the bottom of the plane and you can go to this area, which is supposed to have a door on it before the Skedar Ship attacks Air Force One.

**Comment by Flax4** [comment]
> Go to the room that Trent Easton runs into after he does his evil laugh, go forward and you'll see a trap door to your right. Start in the bottom of the plane and you can go to this area, which is supposed to have a door on it before the Skedar Ship attacks Air Force One.

Yeah, you're gonna have to gather some screenshots or video of it, because I can't see anything wrong.

**Comment by kronflux** [comment]
Yeah this is an isolated issue if you're having one, here's screenshots of before the crash, after the crash the door disappears.
![image](https://github.com/fgsfdsfgs/perfect_dark/assets/834451/8bcaa93b-3f4c-47ca-91f3-fae7c6a1f6a0)

**Comment by PeeMoldy** [comment]
> Yeah this is an isolated issue if you're having one, here's screenshots of before the crash, after the crash the door disappears. ![image](https://private-user-images.githubusercontent.com/834451/294645004-8bcaa93b-3f4c-47ca-91f3-fae7c6a1f6a0.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDQ1NDMyMDQsIm5iZiI6MTcwNDU0MjkwNCwicGF0aCI6Ii84MzQ0NTEvMjk0NjQ1MDA0LThiY2FhOTNiLTNmNGMtNDdjYS05MWYzLWZhZTdjNmExZjZhMC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMTA2JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDEwNlQxMjA4MjRaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1hNjFhYzA4OGEzYjZlZDIwNGI0ZjdmNTM0MDBkNGUxYmViZTcxYjVhNjA3OGY1MDNmODk5NGFlNTRkZmJiNWQ4JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.OpKJsO9m_0ARb9QczsVdd4iB9PaOkhKle99jSVPSr0Y)

Looks like that initial issue has been fixed! But there is still this texture issue...

![pdpcptexturebugaf1](https://github.com/fgsfdsfgs/perfect_dark/assets/152383944/fffcfee7-1277-4c32-a024-2bb1114b4f3d)

**Comment by RyanDwyer** [comment]
That door is blown off at the start of the level on co-op mode when using simulant buddies. My guess is that simulants have problems opening the floor hatch so they chose to remove it. So if you were playing on co-op then that would explain it.

The texture is a bug though.

**Comment by PeeMoldy** [comment]
> That door is blown off at the start of the level on co-op mode when using simulant buddies. My guess is that simulants have problems opening the floor hatch so they chose to remove it. So if you were playing on co-op then that would explain it.
> 
> The texture is a bug though.

Yes this is why it was happening! Thank you. I've noticed that texture since the beginning and I had to say something. It looks glitched in a bad way.

**Comment by Flax4** [comment]
Seems to be a vanilla texture glitch

RMG emulator w/ parallel plugin
![bild](https://github.com/fgsfdsfgs/perfect_dark/assets/50862920/cf72a021-d736-45da-8508-4bee0d3cd4d8)

**Comment by PeeMoldy** [comment]
> Seems to be a vanilla texture glitch
> 
> RMG emulator w/ parallel plugin ![bild](https://private-user-images.githubusercontent.com/50862920/294707655-cf72a021-d736-45da-8508-4bee0d3cd4d8.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDQ1NjYwNDUsIm5iZiI6MTcwNDU2NTc0NSwicGF0aCI6Ii81MDg2MjkyMC8yOTQ3MDc2NTUtY2Y3MmEwMjEtZDczNi00NWRhLTg1MDgtNGJlZTBkM2NkNGQ4LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAxMDYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMTA2VDE4MjkwNVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTI4NGE2MWEyYWJhMjNjNDkwZTdlOWM2YjRiZWFhNjQ2OThmZTUzN2IzNzI5NTg4NDRiYTY3MjRjNWQ4MmEzNDgmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.SOg6F5A6-5kWdzyqAu3rEOzkDeud84R1E3GOTUj9H6I)

You test it on console yet?

**Comment by Flax4** [comment]
> You test it on console yet?

Nintendo 64 emulation isn't perfect, but for purposes like this it suffices. So no, I haven't and I won't. Feel free to do yourself.

**Comment by PeeMoldy** [comment]
> > You test it on console yet?
> 
> Nintendo 64 emulation isn't perfect, but for purposes like this it suffices. So no, I haven't and I won't. Feel free to do yourself.

I'll check it out and report back here.
